.github/workflows/c-cpp.yml

```diff
@@ -13,14 +13,13 @@ jobs:
     - name: Install python
       run: sudo apt-get --assume-yes install python3-wheel python3-setuptools python3-dev ninja-build
     - name: Install meson and ninja
-      run: pip3 install --user meson PyYAML
+      run: sudo pip3 install meson PyYAML ninja
     - uses: actions/checkout@v2
     - name: rizin
       run: |
-          git clone --recursive --depth=1 https://github.com/rizinorg/rizin
+          git clone --depth=1 https://github.com/rizinorg/rizin
```

Comment:
do we want to have stable here?

Reply:
For consistency we may use the same approach everywhere

Reply:
which is?

Reply:
@wargio not for now. We could do the switch after 0.3.0 is out, and RzLang properly fixed.